### PR TITLE
P0 card

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go 🚀

File: css/components/card.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+/* CONTENT INSIDE CARD CHILD */
+.card-content-head,.card-content-body,.card-content-foot{
+  display: flex;
+  flex-flow: row wrap;
+  width:100%;
+  gap:0.5rem;
+}
+
+.space{
```
Comment (sanjitsarkar) [inline css/components/card.css]: Make a separate utility css file for these utility classes

**Review by sanjitsarkar — APPROVED** [review]

File: css/components/card.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  position: relative;
+  display: grid;
+  gap: 0.5rem 0.5rem;
+}
+
+.card-content{
+  display:flex;
+  padding:0.5rem;
+  justify-content: flex-start;
+  align-items: flex-start
```
Comment (dev-ashishsethi) [inline css/components/card.css:12]: semicolon missing

**Review by dev-ashishsethi — APPROVED** [review]
looks good

**Pull request merged**